lib/tapioca/gemfile.rb

```diff
@@ -67,11 +25,14 @@ def autorequire
 
     sig { params(exclude: T::Array[String]).void }
     def initialize(exclude)
-      AutoRequireHook.exclude = exclude
       @gemfile = T.let(File.new(Bundler.default_gemfile), File)
       @lockfile = T.let(File.new(Bundler.default_lockfile), File)
       @definition = T.let(Bundler::Dsl.evaluate(gemfile, lockfile, {}), Bundler::Definition)
-      dependencies, missing_specs = load_dependencies
+
+      dependencies, missing_specs = BundlerExt::AutoRequireHook.override_require_false(exclude: exclude) do
+        load_dependencies
+      end
```

Comment:
What do you think about moving the `override_require_false` inside `load_dependencies`?

Reply:
Ah good call! It also turns out that the call should be made inside `require_bundle` call anyway. 🙈